### ISSUE TITLE
[BUG] plugin.tx_powermail.settings.setup.receiver.overwrite.replyToEmail.value is ignored

### ISSUE THREAD

**Issue by glucka** [in2code-de/powermail#557]
The settings 
```
plugin.tx_powermail.settings.setup.receiver.overwrite.replyToEmail.value 
plugin.tx_powermail.settings.setup.receiver.overwrite.replyToName.value 
```
is ignored in v7 und v8
I make a PR now.

**Comment by MrMooky** [comment]
This is still the case in TYPO3 9.5.19 with Powermail 7.4.0. Basically, the constants are correctly set (as per TypoScript Object Browser), but these values are not forwarded when sending the email.

`sendMail(array('template' => 'Mail/OptinMail', 'receiverEmail' => 'me@domain.de', 'receiverName' => 'Peter Griffin', 'senderEmail' => 'noreply@domain.de', 'senderName' => 'Test', 'replyToEmail' => '', 'replyToName' => '', 'subject' => 'Bitte bestätigen Sie Ihre Emailadresse', 'rteBody' => '', 'format' => 'both', 'variables' => array('hash ....`

**Comment by MrMooky** [comment]
The PR suggested by @glucka is working, but this should be made available in a TYPO3 9 compatible version. It appears the bug has been fixed in Powermail 8.0.0, which is only compatible with TYPO3 v10.

**Comment by kurtkk** [comment]
What shall we do now with powermail in TYPO3 V9!?
Wait for a bugfix?
Wait for a powermail-upgrade for TYPO3 V9?

**Comment by MrMooky** [comment]
Just implement the [Commit](https://github.com/glucka/powermail/commit/1050f0718d639e3ee7e1691cee41700f2f0fb13b) mentioned above.

**Comment by Teisi** [comment]
@MrMooky can not confirm that this work for TYPO3 9....

**Comment by glucka** [comment]
@Teisi @MrMooky I use this in v9

**Comment by MrMooky** [comment]
@Teisi Worked as expected in our case with TYPO3 9.5.19 and Powermail 7.4.0

**Comment by Teisi** [comment]
@MrMooky what have you done exactly? I have copied and replaced the complete method "parseAndOverwriteVariables"...

**Comment by MrMooky** [comment]
@Teisi I only implemented the new lines mentioned [here](https://github.com/glucka/powermail/commit/1050f0718d639e3ee7e1691cee41700f2f0fb13b), nothing else.

**Comment by Teisi** [comment]
@MrMooky hmm, after the change, i'm not getting any e-mails at all... strange. I have to look closer...
thank you.

**Comment by kurtkk** [comment]
The code of the commit above does not exist in Powermail V7.4.0  **/powermail/Classes/Domain/Service/MailSendMailService.php** !?

**Comment by einpraegsam** [comment]
I just tested powermail 7.4.0 (with latest TYPO3 9) and powermail 8.2.4 (with latest TYPO3 10) with this test settings:
```
plugin.tx_powermail.settings.setup.receiver.overwrite.replyToEmail = TEXT
plugin.tx_powermail.settings.setup.receiver.overwrite.replyToEmail.value = test@in2code.de
plugin.tx_powermail.settings.setup.receiver.overwrite.replyToName = TEXT
plugin.tx_powermail.settings.setup.receiver.overwrite.replyToName.value = test
```

The generated mail has this header:
```
Content-Type: multipart/alternative; boundary="_=_swift_v4_1607004812_5ae41346f23f0c89ad8d4cc07d18f88d_=_"
Date: Thu, 03 Dec 2020 15:13:32 +0100
From: defaultmailfromaddress <alexander.kellner@einpraegsam.net>
MIME-Version: 1.0
Message-ID: <626edb3fc09f52658672c4217fd71dde@powermail95.localhost.de>
Received: from powermail95.localhost.de by mailhog.example (MailHog)

          id TLwAk23GCK-kBa9qvoPZcebvrwvGdxc7B7GrKTkzUUY=@mailhog.example; Thu, 03 Dec 2020 15:13:32 +0100
Reply-To: test <test@in2code.de>
Return-Path: <alexander.kellner@einpraegsam.net>
Subject: [testparcours] New Email
To: Alex <alexander.kellner@in2code.de>
X-Mailer: TYPO3
X-Priority: 3 (Normal)
```

The question is: How can I reproduce the issue?

**Comment by glucka** [comment]
Maybe only if $GLOBALS['TYPO3_CONF_VARS']['MAIL']['defaultMailReplyToAddress'] is defined?

**Comment by christophbee** [comment]
After entering a mail address into $GLOBALS['TYPO3_CONF_VARS']['MAIL']['defaultMailReplyToAddress'] it works fine
Thx @glucka

**Comment by mschwemer** [comment]
Needs further investigation together with the other email related stuff

**Comment by mschwemer** [comment]
Citing a comment `It appears the bug has been fixed in Powermail 8.0.0, which is only compatible with TYPO3 v10.`

So we are now on TYPO3 v12 and Powermail v12, I close this issue and the mentioned PR.

If you are still hit by this issue, please re-open it. Thx.
